frontend/package.json

```diff
@@ -24,7 +24,8 @@
     "bnc-onboard": "^1.13.0",
```

Comment:
When I run `yarn` at the top level, my `yarn.lock` ends up modified. Let's check in any diffs that exist

Reply:
This happened to me also. @garyghayrat I recommend running `yarn` from the project root and commit the resulting lockfile (assuming it changes)

Reply:
@mds1 Updated yarn.lcok, try now.

Reply:
Perfect, thanks!